mergify_engine/sub_utils.py

```diff
@@ -121,9 +122,10 @@ def _retrieve_subscription_from_cache(r, installation_id):
         return _decrypt(encrypted_sub)
 
 
-def save_subscription_to_cache(r, installation_id, sub):
+async def save_subscription_to_cache(installation_id, sub):
+    r = await utils.get_aioredis_for_cache()
     encrypted = _encrypt(sub)
-    r.set("subscription-cache-%s" % installation_id, encrypted, ex=3600)
+    await r.setex("subscription-cache-%s" % installation_id, 3600, encrypted)
 
 
 def get_subscription(r, installation_id):
```

Comment:
TODO async I guess?

Reply:
A lot are TODO for async, method will migrate slowly :)